src/mgr/MgrClient.cc

```diff
@@ -31,6 +31,12 @@
 #include "messages/MMgrCommandReply.h"
 #include "messages/MPGStats.h"
 
+#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
```

Comment:
The manager isn't my area, but I can approve if you drop this ifdef here and in `mgr_perf_counters.cc` with the same rationale as for RBD in the other PR.

Reply:
Aye, removed.